components/blocks/tile.module.css

```diff
@@ -65,3 +65,26 @@
 :global(.dark) .TransparentBackground p {
   @apply text-white !important;
 }
+.YellowBackground i,
+.YellowBackground h4{
+  @apply text-[#EAA623] !important;
+}
+
+.VioletBackground i,
+.VioletBackground h4{
+  @apply text-lightBlue-70 !important;
+}
+
+.PurpleBackground i,
+.PurpleBackground h4{
+  @apply text-color_violet !important;
```

Comment:
Same here. Looks like this inconsistency is almost everywhere. Please fix it all places.